src/coprocessor/statistics/cmsketch.rs

```diff
@@ -30,12 +29,8 @@ impl CmSketch {
 
     // `hash` hashes the data into two u64 using murmur hash.
     fn hash(mut bytes: &[u8]) -> (u64, u64) {
-        let mut out: [u8; 16] = [0; 16];
-        murmur3_x64_128(&mut bytes, 0, &mut out);
-        (
-            LittleEndian::read_u64(&out[0..8]),
```

Comment:
The order may be incorrect due to different endianness.

Reply:
PTAL :rofl:

Reply:
After checking the source of `murmur3_x64_128`, the correct way should be "(out as u64, (out >> 64) as u64)". I used to think it use `u128::from_le` to compute the value, turns out it uses shift and or to combine two u64 into a u128.

Reply:
It was my mistake. After comparing the fork version and `v0.5.1` carefully, I think it should be right now.